### DISCUSSION TITLE
Sorting by book date

### DISCUSSION THREAD

**Discussion by karma-riuk** [firefly-iii/firefly-iii#5197] [Firefly III]
Hello :)

First of all, I want to thank you for your awesome software, I really think it is great!

My question is the following:
Is there a way to set the book date as the sorting date while looking at transactions? Because it happens very often (almost always) that my bank saves a certain transaction 1 to 4 working days after the transaction actually occurred and it becomes quite complicated when I try to create a reconciliation or make balances match on a specific date.

The only solution to this issue that I thought of was to invert the dates (i.e. make the date of the transaction have the same date as the bank and set the book date as the date where the transaction actually occurred), but this solution frustrates me a bit, since the transactions won't be in the correct chronological order when looking at the transactions table.

Would you have any suggestions to help me?

Thank you in advance for your help :)

**Comment by JC5** [comment]
This isn't possible I'm afraid. I'm looking into ways to improve this in the new layout but don't hold your breath.
